src/com/owncloud/android/lib/resources/files/ChunkedUploadRemoteFileOperation.java

```diff
@@ -99,6 +100,12 @@ protected int uploadFile(OwnCloudClient client) throws IOException {
                 mPutMethod.addRequestHeader(OC_CHUNKED_HEADER, OC_CHUNKED_HEADER);
                 mPutMethod.addRequestHeader(OC_CHUNK_SIZE_HEADER, chunkSizeStr);
                 mPutMethod.addRequestHeader(OC_TOTAL_LENGTH_HEADER, totalLengthStr);
+
+                // Tell to the server what is the last modification date of the file to upload
+                Long timeStampLong = System.currentTimeMillis()/1000;
+                String timeStamp = timeStampLong.toString();
+                mPutMethod.addRequestHeader(OC_CHUNK_X_OC_MTIME_HEADER, timeStamp);
+
```

Comment:
Let's take this PR as a chance to fix the format of these two modified files. Please, replace all the tabs in them with whitespaces (4 whitespaces per tab). Thanks.